### PR TITLE
[staging] Increased smoketest timeout

### PR DESCRIPTION
**Pull request by sutt0n**
Increasing the `defaultCommandTimeout` for cypress smoketests from 12000ms to 20000ms due to a theory on the env is freshly deployed (along with the monolambda), so when the endpoint(s) are hit for the first time, and the lambda is cold, it's timing out -- or an operation is being slow.

### CODE REVIEW

File: cypress-smoketests.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://ui-dev.ustc-case-mgmt.flexion.us",
+  "baseUrl": "https://ui-stg.ustc-case-mgmt.flexion.us",
```
Comment (rachaelparris) [inline cypress-smoketests.json]: It's really not good that this is hardcoded. Is now the time to fix that?

File: cypress-smoketests.json [hunk 2]
```diff
@@ -10,7 +9,7 @@
   "screenshotsFolder": "cypress-smoketests/screenshots",
   "videosFolder": "cypress-smoketests/videos",
   "supportFile": "cypress-smoketests/support/index.js",
-  "defaultCommandTimeout": 20000,
+  "defaultCommandTimeout": 12000,
```
Comment (rachaelparris) [inline cypress-smoketests.json]: Did you mean to put this back to 12000?

Comment (sutt0n) [reply cypress-smoketests.json]: *le sigh*

Comment (sutt0n) [reply cypress-smoketests.json]: nope

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**